### ISSUE TITLE
Make iOS touch interactions feel more native

### ISSUE THREAD

**Issue by adamschwartz** [EagerIO/EmbedBox#78]
Right now everything feels webby and not native. We can improve this with a few things:

- Make the `-webkit-tap-highlight` transparent.
- Handle touchstart as click or use [the touchstart trick as seen e.g. in LeadLine](https://github.com/EagerIO/LeadLine/blob/0cbc8422bb229bff0ce3ca154267d3d7d88ca746/eager-lead-line.js#L46).
- Fix animation jitters.

**Comment by TeffenEllis** [comment]
@adamschwartz,

This one was something else! 😅 

It seems fixed position elements and iOS Safari don't play well. The fix involved switching to a position absolute container for small screens and a few viewport tricks. The heaviest part of these fixes is the tracking of the body's scroll state. Safari loses this when setting overflow hidden on the `html` element. Unfortunately this seems to be the patch that satisfies the collection of jitteriness and behavior bugs we've encountered. 

The commit that closes this issue addresses all but the touchstart click feature. I've tried the feature on a dev server with mixed success. 

If the user scrolls the search list with their finger over a entry, it will momentarily highlight.
If the highlight occurs during a transition between screens, It's often accompanied by a slow down.

Cheers
